CHANGELOG.latest.md

```diff
@@ -1 +1,34 @@
-- Removed deprecated `addAttributionData` functions and `AttributionNetwork` enum.
\ No newline at end of file
+**Note:** This release is based off of 5.0.0-rc3. These are the changes compared to 5.0.0-rc3:
+
+- Removed deprecated `addAttributionData` functions and `AttributionNetwork` enum
+    https://github.com/RevenueCat/purchases-android/pull/506
+- Added retries with exponential backoff when reconnecting to Google's billing service
+    https://github.com/RevenueCat/purchases-android/pull/489
+- Updated to BillingClient 4.1
+    https://github.com/RevenueCat/purchases-android/pull/500
+- Prevent Amazon from swallowing exceptions
+    https://github.com/RevenueCat/purchases-android/pull/499
+- Improvements on Amazon implementation preventing a lot of issues related to price and currency parsing
+    https://github.com/RevenueCat/purchases-android/pull/503
+
+[Full Changelog](https://github.com/revenuecat/purchases-android/compare/main...5.0.0)
+
+### Migration Guide
+- A number of renames, removal of some deprecated functions, and a change to configuration. Amazon Store support is also included Checkout the full list of changes in [migrations/v5-MIGRATION.md]
+
+### API changes:
+- See our [RevenueCat V5 API update doc]([migrations/v5-MIGRATION.md]) for API updates.
+
+#### Amazon Support
+**[Experimental]** Introduced support for using Amazon Store. This is currently in experimental phase, and there could be issues, please report any issue you find. [New RevenueCat Issue](https://github.com/RevenueCat/purchases-android/issues/new/).
```

Comment:
do we still consider this to be experimental?

Reply:
Hmmm yeah, it's not experimental. I don't know how to call it. I want to make sure they understand there might be some bugs

Reply:
Perhaps we should very explicitly call out the part(s) that might have some issues or awkwardness